apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -188,10 +181,14 @@ class ManageStudentsTable extends Component {
         }
         {numberOfEditingRows <= 1 &&
           <span>
-            {i18n.actions()}
-            {showShareColumn &&
-              <FontAwesome icon="cog" style={styles.cog}/>
-            }
+            <div style={{width: '40%', float: 'left'}}>
```

Comment:
- [ ] Pull these inline styles into a styles object that can be shared between the two